src/Catty/PlayerEngine/Sensors/SensorManager.swift

```diff
@@ -27,17 +27,23 @@ import CoreMotion
 
     public static var defaultValueForUndefinedSensor: Double = 0
     private static var sensorMap = [String: Sensor]() // TODO: make instance let
+    var landscapeMode: Bool = false
```

Comment:
Is it necessary to change this value after initialization? If not, please change to `let`. Also change scope to `private`.

Reply:
No it was not necessary and your suggestion is the proper way, thanks.